spring-cloud-vault-config/src/main/java/org/springframework/cloud/vault/config/VaultHealthIndicator.java

```diff
@@ -45,7 +49,7 @@ public Health health() {
 			}
 
 			if (vaultHealthResponse.isStandby()) {
-				return Health.outOfService().withDetail("state", "Vault in standby")
+				return (standbyOk ? Health.up() : Health.outOfService()).withDetail("state", "Vault in standby")
```

Comment:
Let's just keep the change to `Health.up()` with the detail message and remove all other changes to accept standby as healthy state.